xml/System/ReadOnlySpan`1.xml

```diff
@@ -727,7 +727,7 @@ This reference is used for pinning and must never be dereferenced.]]></format>
         <summary>Copies the contents of this read-only span into a new array.</summary>
         <returns>An array containing the data in the current span.</returns>
         <remarks>
-          <format type="text/markdown"><![CDATA[This method performs a heap allocation and data copy, and therefore, should be avoided if possible. It it sometimes necessary when APIs that support <xref:System.ReadOnlySpan%601> objects are not available, but APIs that work with arrays are.]]></format>
+          <format type="text/markdown"><![CDATA[This method performs a heap allocation and data copy, and therefore, should be avoided if possible. It it sometimes necessary when APIs that support <xref:System.ReadOnlySpan%601> value types are not available, but APIs that work with arrays are.]]></format>
```

Comment:
nit: this is awkward to read, imo, and could cause confusion (are we talking about the T being the value type or referring to Span itself as the value type).

How about removing `value types` all together:
"It is sometimes necessary when APIs that support `<xref:System.ReadOnlySpan%601>` are not available, ..."

Reply:
Yeah, I was trying to make it plural. I'm not sure how to word that.

Reply:
@ahsonkhan I appreciate you looking at this 😄 I changed the "value types" to "instances" which is used in other struct equals docs.